src/sentry/integrations/vercel/integration.py

```diff
@@ -42,7 +43,32 @@
 
 
 class VercelIntegration(IntegrationInstallation):
-    pass
+    def get_organization_config(self):
```

Comment:
is there a reason not to have `update_organization_config` and `get_config_data` in this PR? If so, can we update the jira ticket criteria? Since right now this only displays the project mapping, it doesn't save the data or populate with an already existing configuration. (not sure if part II finishing touches means just updating the UI)

Reply:
@MeredithAnya We don't need to override those methods from the base class in order to save the project mappings. That's already happening.

Reply:
ahh I just assumed those both raised NotImplementedError hokays cool